server/portal/apps/projects/management/commands/migrate-projects.py

```diff
@@ -0,0 +1,51 @@
+"""Management command."""
+
+from django.core.management.base import BaseCommand
+from portal.libs.agave.utils import service_account
+from portal.apps.projects.models.base import Project
+from portal.apps.projects.models.metadata import AbstractProjectMetadata
```

Comment:
I think the `ProjectMetadata` model should be imported/used here, since in theory (at least in my understanding) abstract classes shouldn't be directly instantiated.

Reply:
In this case, it shouldn't. The ProjectMetadata class is has no fields, therefore no migration and no corresponding objects are created for it in the database.